Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -275,6 +275,7 @@ public extension CBPurchase {
                             completion?(.failure(CBError.defaultSytemError(statusCode: 400, message: "Invalid Purchase")))
                             return
                         }
+                        self.activeProduct = nil
```

Comment:
Can we have a test around this?

Reply:
As discussed, we can cover this as part of next change.

Reply:
Yes Sure will cover UnitTest on next upcoming changes due to high priority we are releasing this.